ChatWithAuthSample/AuthorizationSample/Dockerfile

```diff
@@ -0,0 +1,23 @@
+#Depending on the operating system of the host machines(s) that will build or run the containers, the image specified in the FROM statement may need to be changed.
+#For more information, please see https://aka.ms/containercompat
+
+FROM microsoft/dotnet:2.1-aspnetcore-runtime-nanoserver-1803 AS base
```

Comment:
We're close to releasing .NET Core 3.0. Would you be interested in upgrading the app to 3.0 preview 7? I think that'd provide more value.

Reply:
I can give it a go this weekend and let you know if it all still works.

Reply:
I updated it to .NET Core 3.0 Preview 6 and checked it in. I started on the Angular app which I'll use to test and verify. Will get back to you all once I can confirm it is good to go.

Reply:
Tested/working with the update to .NET Core 3.0 Preview 6. Angular 7 App:
https://github.com/h5aaimtron/sample_chatWithIdentityClient

Reply:
Why is this a windows container

Reply:
Defaulted docker to windows at the time. I switch back and forth. I can update it as needed.